key.ui/src/main/java/de/uka/ilkd/key/gui/TaskTree.java

```diff
@@ -253,10 +249,13 @@ public void mousePressed(MouseEvent e) {
 
         private void checkPopup(MouseEvent e) {
```

Comment:
```suggestion0
        @Override
        public void mouseReleased(MouseEvent e) {
            checkPopup(e);
        }

        /**
         * Checks whether the popup menu should be shown and does so if necessary.
         * <br>
         * <b>Important:</b><br>
         * For the platform specific popup trigger to work, we need to check the popup in pressed, released, and clicked
         * event. For example, on Windows the e.isPopupTrigger() information is only available in the released event.
         *
         * @param e the mouse event that may create the popup
         */
         private void checkPopup(MouseEvent e) {
```
Without this code, the popup menu does not work on Windows (isPopupTrigger() call only returns true in mouseReleased!). Can you add this please?